docs/mixin/dashboards/promscale.json

```diff
@@ -761,9 +761,9 @@
                 "uid": "${DS_PROMETHEUS}"
               },
               "exemplar": true,
-              "expr": "rate(grpc_server_handled_total{grpc_service=~\"opentelemetry.proto.collector.trace.v1.TraceService\",grpc_method=~\"Export\"}[$__rate_interval]) > 0",
+              "expr": "sum(rate(grpc_server_handled_total{grpc_service=~\"opentelemetry.proto.collector.trace.v1.TraceService\",grpc_method=~\"Export\"}[$__rate_interval])) + sum(rate(grpc_server_msg_received_total{grpc_service=\"jaeger.storage.v1.StreamingSpanWriterPlugin\",grpc_method=\"WriteSpanStream\"}[$__rate_interval])) + sum(rate(grpc_server_handled_total{grpc_service=\"jaeger.storage.v1.SpanWriterPlugin\",grpc_method=\"WriteSpan\"}[$__rate_interval]))",
```

Comment:
I have very little experience with PromQL so I don't know if there's a better way to add the 3 metrics